apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -53,9 +53,29 @@ const LocationInput = (props: {
 }): JSX.Element | null => {
   const { eventLocationType, locationFormMethods, ...remainingProps } = props;
   const { control } = useFormContext() as typeof locationFormMethods;
+
   if (eventLocationType?.organizerInputType === "text") {
+    const { defaultValue, ...rest } = remainingProps;
+
     return (
-      <Input {...locationFormMethods.register(eventLocationType.variable)} type="text" {...remainingProps} />
```

Comment:
Can we render this conditionally? That way this won't break for people who aren't opting to use autocomplete.